packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -656,9 +656,9 @@ class RenderConstrainedOverflowBox extends RenderAligningShiftedBox {
   }
 }
 
-/// A [RenderBox] that applies an arbitrary transform to its [constraints]
-/// before sizing its child using the new constraints, treating any overflow as
-/// error.
+/// A [RenderBox]  that applies an arbitrary transform to its constraints,
```

Comment:
looks like an extra space snuck in here.

Reply:
1) already did, in https://github.com/flutter/flutter/pull/110393
2) already fixed.